charts/falco/values-gvisor-gke.yaml

```
@@ -57,11 +55,11 @@ falcoctl:
         resolveDeps: false
         # -- List of artifacts to be installed by the falcoctl init container.
         # We do not recommend installing (or following) plugins for security reasons since they are executable objects.
-        refs: [falco-rules:1]
+        refs: [falco-rules:2]
       follow:
         # -- List of artifacts to be followed by the falcoctl sidecar container.
         # We do not recommend installing (or following) plugins for security reasons since they are executable objects.
-        refs: [falco-rules:1]
+        refs: [falco-rules:2]
```

```suggestion
        refs: [falco-rules:3]
```
Just a reminder: rules 3.0 are not yet release, but they will be soon.